src/RegisterHandler.php

```diff
@@ -259,12 +256,10 @@ protected function getCredentialCreationOptions(
             $this->getUserEntity($store->getMember()),
             random_bytes(32),
             [new PublicKeyCredentialParameters('public-key', Algorithms::COSE_ALGORITHM_ES256)],
-            40000,
-            [],
-            $this->getAuthenticatorSelectionCriteria(),
-            PublicKeyCredentialCreationOptions::ATTESTATION_CONVEYANCE_PREFERENCE_NONE,
-            new AuthenticationExtensionsClientInputs()
+            40000
```

Comment:
Do we know what that `40000` number represent? Might be worth putting it in a constant for clarity.

Reply:
$timeout - Webauthn\PublicKeyCredentialCreationOptions::construct()
presumably 40 seconds